paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -140,7 +139,7 @@ You can run `test_analyzer_int8_object_detection` with the following arguments t
 
 ```bash
 cd /PATH/TO/PADDLE/build
-./paddle/fluid/inference/tests/api/test_analyzer_int8_object_detection --infer_model=third_party/inference_demo/int8v2/mobilenet-ssd/model --infer_data=$HOME/.cache/paddle/dataset/pascalvoc/pascalvoc_full.bin --warmup_batch_size=10 --batch_size=100 --paddle_num_threads=1
+./paddle/fluid/inference/tests/api/test_analyzer_int8_object_detection --infer_model=third_party/inference_demo/int8v2/mobilenet-ssd/model --infer_data=$HOME/.cache/paddle/dataset/pascalvoc/pascalvoc_full.bin --warmup_batch_size=10 --batch_size=1 --paddle_num_threads=1
 ```
```

Comment:
When batch_size = 100, Ratio(INT8/FP32) = 3.04 ; When batch_size = 1, Ratio(INT8/FP32) = 1.27. We will give more explanations tomorrow.